### PR TITLE
Team Section

### PR DESCRIPTION
**Pull request by snwessel**
Creating the **Our Team** section (by both Nupur and I)
- Pulls data from Airtable 
- Filters based on what is selected

A couple quick notes: 
- I changed the font for the _entire_ livesite to Nunito in this PR, but I would be happy to move that change to a separate PR if anyone thinks I should. 
- The photo for the full team will be replaced with our core team hype video once it's done.

Desktop:
image
image

Mobile: 
image
image

### CODE REVIEW

File: src/styles/team.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.test {
```
Comment (nupur-n) [inline src/styles/team.scss]: do we need this one

File: src/styles/team.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  .selected {
+    color: white;
+    background-color: $orange;
+    border-width: 0;
+    box-shadow: none;
+  }
+  
+  &__card {
```
Comment (nupur-n) [inline src/styles/team.scss]: is there global card styling that we are using on all cards throughout our site? wondering if these & mentors are going to be the same or if it makes sense to have their styling separate

File: src/components/Team.js [hunk 3]
```diff
@@ -1,14 +1,88 @@
...
+        </button>
+        <button className={`btn btn-light teams__button ${(team==='sponsorship')?'selected':''}`} type="button" onClick={() => setTeam('sponsorship')}>
+          Sponsorship Team
+        </button>
+      </div>
+    </Row>
+    <Container>
+      <Row className="py-3">
+          {isLoading && <Spinner animation='border' variant='primary' /> /* TODO import custom styled spinner component*/}
+          {!data.length && <div>Nothing to see here...</div> /* TODO handle empty array in case of API error (or if its actually empty for any reason)*/}
```
Comment (daniaalnahdi) [inline src/components/Team.js]: I forgot to do this initially (my bad!), the condition should be `!data.length && !isLoading`

**Review by daniaalnahdi — CHANGES_REQUESTED** [review]

File: src/styles/team.scss (hunk 1 above)
Comment (snwessel) [reply src/styles/team.scss]: Oops, no. Good catch!

File: src/styles/team.scss (hunk 2 above)
Comment (snwessel) [reply src/styles/team.scss]: Oh thank you! I had meant to make one central card styling definition, but I forgot

**Comment by snwessel** [comment]
Thanks for reviewing @daniaalnahdi and @nupur-n 😀  I just updated with your suggestions!

**Review by nupur-n — APPROVED** [review]
Looks great!! woooo!

**Review by daniaalnahdi — APPROVED** [review]

**Pull request merged**